src/style-spec/reference/v8.json

```diff
@@ -3345,6 +3355,17 @@
           }
         }
       },
+      "distance": {
+        "doc": "Returns the shortest distance in meters between the evaluated feature and the input geometry. The evaluated feature and input value can be a valid GeoJSON of type `Point`, `MultiPoint`, `LineString`, `MultiLineString`, `Polygon`, `MultiPolygon`, `Feature`, or `FeatureCollection`. Results at zoom level 12 and below may vary in precision due to precision loss in conversions between tile coordinates and `[Longitude, Latitude]` at lower zoom levels.",
+        "group": "Math",
```

Comment:
What do you think of adding a new `Geospatial` group for `within` and `distance`?

Reply:
I think we can always do this later if we find ourselves adding more geospatial expressions. Generally I'm hesitant to create a new category for fewer than 3 members, and `within` fits in the Decision group pretty well.  I agree, `distance` is a bit more awkward in the Math group, but I think it works fine enough.